analysis/wwz/make_datacards.py

```diff
@@ -130,7 +134,11 @@ def make_ch_card(ch,proc_order,ch_ylds,ch_kappas=None,ch_gmn=None,out_dir="."):
 #   - Because of how we fill in the processor, the yields for per year systs come _only_ from that year
 #   - So this function adds the nominal yields from the other three years to the up/down variation for the relevant year
 #   - Note the in_dict is modifed in place (we do not return a copy of the dict)
-def handle_per_year_systs_for_fr2(in_dict, systs_special=SYSTS_SPECIAL):
+def handle_per_year_systs_for_fr2(in_dict,year):
```

Comment:
Should rename. The "fr2" was for full run 2. I would just drop the 2 from the name.